browser/scripts/loaders/assetLoader.js

```diff
@@ -0,0 +1,176 @@
+(function() {
+
+if (typeof(module) !== 'undefined') {
+	EventEmitter = require('events').EventEmitter
+	when = require('when')
+}
+
+var loadingPlugins = {
+	'three_loader_model': 'model',
+	'three_loader_scene': 'scene',
+	'url_texture_generator': 'texture',
+	// 'url_audio_buffer_generator': 'audiobuffer',
+	// 'url_audio_generator': 'audio',
+	// 'url_json_generator': 'json',
+	// 'url_video_generator': 'video'
+}
+
+// ------------------------------------------------
+
+function AssetLoader(loaders) {
+	EventEmitter.call(this)
+
+	this.loadingTexture = THREE.ImageUtils.loadTexture('/data/textures/loadingtex.png')
```

Comment:
please do:
	this.defaultTexture.wrapS = THREE.RepeatWrapping
	this.defaultTexture.wrapT = THREE.RepeatWrapping

and the same for loadingTexture to enable repeat wrapping